gradle.properties

```diff
@@ -3,7 +3,7 @@ javafxPlatform=unspecified
 faf_ice_adapter_version=2.3.0
 faf_uid_version=4.0.4
 propdepsVersion=0.0.7
-springBootVersion=2.1.0.RC1
+springBootVersion=2.1.6.RELEASE
```

Comment:
One does not simply switch the spring boot version to fix tests.
If you want to update dependencies, please do at least all of the spring ones at once.

Reply:
Right, i upped the oauth autoconfigure as well to the same version.